src/java.desktop/windows/classes/sun/awt/windows/WFontConfiguration.java

```diff
@@ -79,8 +79,7 @@ protected void initReorderMap() {
         reorderMap.put("GBK", "chinese-ms936");
         reorderMap.put("GB18030", "chinese-gb18030");
         reorderMap.put("x-windows-950", "chinese-ms950");
-        reorderMap.put("x-MS950-HKSCS", split("chinese-ms950,chinese-hkscs"));
-//      reorderMap.put("windows-1252", "alphabetic");
```

Comment:
...Why did we have this? Removing it is fine, but it gives me the "oo we shouldn't touch it just in case" feeling.